### DISCUSSION TITLE
Add API details for `collections.move`

### DISCUSSION THREAD

**Discussion by zeekhuge** [outline/outline#5788] [General] [answered]
The App use `collections.move` API that resets the value of `index`. The API documentation at https://www.getoutline.com/developers#tag/Collections mentions nothing about it.
I wanted to understand the logic of the value that is being set as the `index`.

**Comment by tommoor** [comment, marked as answer]
`index` is a fractional index, you can read more about the [approach we use here](https://observablehq.com/@dgreensp/implementing-fractional-indexing). The main reason for this is that you can insert an item between any other two without having to rewrite the other rows.

If you're using the API you could use the [same NPM module](https://www.npmjs.com/package/fractional-index) to calculate indexes.
